compose.yml

```diff
@@ -6,7 +6,8 @@ services:
       args:
         PHP_VERSION: ${PHP_VERSION:-8.1}
         MAGENTO_VERSION: ${MAGENTO_VERSION:-2.4.6-p6}
-    user: ${UID:-1000}:${GID:-1000}
```

Comment:
I removed this because in Github actions we had a permission error (See the link to the failing CI in the description)
From what I understood from the debug I added, github action uses a user `runner` with UID `1001`, and with this statement, the user `phpuser` was created in the docker with the UID `1000` (It was apparently using the default value)
I see [here](https://blog.giovannidemizio.eu/2021/05/24/how-to-set-user-and-group-in-docker-compose/) that this statement does not work well on Linux, but it's from 2021 so it may be outdated

With the changes, the user `phpuser` is correctly mapped with the same UID than the host user `runner` (`1001`) 
Maybe I'm missing something though, we can discuss this together as of course I don't master the subject :)